ui/app/serializers/database/role.js

```diff
@@ -66,10 +66,7 @@ export default RESTSerializer.extend({
 
   serializeAttribute(snapshot, json, key, attributes) {
     // Don't send values that are undefined
-    if (
-      undefined !== snapshot.attr(key) &&
-      (snapshot.record.get('isNew') || snapshot.changedAttributes()[key])
-    ) {
+    if (undefined !== snapshot.attr(key) && (snapshot.record.isNew || snapshot.changedAttributes()[key])) {
```

Comment:
```suggestion
    if (snapshot.attr(key) !== undefined && (snapshot.record.isNew || snapshot.changedAttributes()[key])) {
```

just a nit, but seeing `undefined` first feels confusing to me 🤔

Reply:
I was hesitant to refactor too much of the original code in this PR to avoid scope creep so I didn't even look at the beginning of this conditional 😆 

I can swap the order if this feels like a reasonable clean up task to include here 🧹

Reply:
done!